csm/conf/configure.py

```diff
@@ -134,7 +142,7 @@ def _rsyslog(self):
         os.makedirs(const.RSYSLOG_DIR, exist_ok=True)
         if os.path.exists(const.RSYSLOG_DIR):
             Setup._run_cmd(f"cp -f {const.SOURCE_RSYSLOG_PATH} {const.RSYSLOG_PATH}")
-            Setup._run_cmd("systemctl restart rsyslog")
+            # Setup._run_cmd("systemctl restart rsyslog")
```

Comment:
@udayan-y21 please remove this commented line

Reply:
Okay.

Reply:
Changes done.